runtime/src/lib.rs

```diff
@@ -272,6 +273,21 @@ impl pallet_babe::Config for Runtime {
         pallet_babe::EquivocationReportSystem<Self, Offences, Historical, ReportLongevity>;
 }
 
+parameter_types! {
+    pub const BagThresholds: &'static [u64] = &bag_thresholds::THRESHOLDS;
+}
+
+type VoterBagsListInstance = pallet_bags_list::Instance1;
+type VoteWeight = u64;
+
+impl pallet_bags_list::Config<VoterBagsListInstance> for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type ScoreProvider = Staking;
+    type WeightInfo = weights::pallet_bags_list::ZKVWeight<Runtime>;
+    type BagThresholds = BagThresholds;
+    type Score = VoteWeight;
```

Comment:
Shouldn't this be taken from the election pallet @drgora ?

Reply:
Oh yes, good catch. `VoteWeight` would be better be taken from `sp-npos-elections`, which we already import indirectly.